index.html

```diff
@@ -30,6 +30,7 @@
 </nav>
 </header>
   <section class="body-section body-section1">
+    <div class="bsm">
       <h1 class="item1 h1">I’m Kirubel. Glad to see you!</h1>
         <li class="item1 p"><p>I’m a software developer! I can help you build a product , 
             feature or website Look through some of my work and experience! 
```

Comment:
:orange_circle: [OPTIONAL] Kindly make sure you indent your logo properly on your mobile version to match the Figma Design. Please consider adding more room around  the page  content  to reflect closely the Figma Design

Your Design
![indentation](https://user-images.githubusercontent.com/26736582/229287421-81e00f04-d8a7-48bb-bc29-53f33d4da4da.png)


Figma Design

![figmaspacing](https://user-images.githubusercontent.com/26736582/229287427-ef764bbd-eaaf-48f0-b04a-88ffd32ee9c4.png)